traffic_ops/traffic_ops_golang/tenant/tenancy.go

```diff
@@ -112,7 +112,42 @@ func GetUserTenantList(user auth.CurrentUser, db *sqlx.DB) ([]Tenant, error) {
 		if err := rows.Scan(&tenantID, &name, &active, &parentID); err != nil {
 			return nil, err
 		}
-		tenants = append(tenants, Tenant{ID: tenantID, Name: name, Active: active, ParentID: parentID})
+		if parentID != nil {
+			tenants = append(tenants, Tenant{ID: tenantID, Name: name, Active: active, ParentID: *parentID})
+		} else {
+			tenants = append(tenants, Tenant{ID: tenantID, Name: name, Active: active, ParentID: -1})
```

Comment:
Agreed, this is why I called it out in the pr comment because I didn't like it either. This function isn't even used yet as I switched to pulling only the IDs anyway.